include/abb_librws/v1_0/rw/retcode.h

```diff
@@ -0,0 +1,40 @@
+#include <abb_librws/rws.h>
+#include <abb_librws/v1_0/rws_client.h>
+
+#include <string>
+
+
+namespace abb :: rws :: v1_0 :: rw
+{
+    using namespace rws::rw;
+}
+
+
+namespace abb :: rws :: v1_0 :: rw :: retcode
+{
+    /**
+     * \brief A structure for containing response of recode resource.
+     */
+    struct RetcodeInfo
+    {
+        int code;                   /** \brief The error code or success code as number */
+        std::string name;           /** \brief The name of error code or success code */
+        std::string severity;       /** \brief Success, Warning or Error */
+        std::string description;    /** \brief Short description of the error code */
+    };
+
+
+    /**
+     * \brief A function for checking description of diagnostic code.
+     *
+     * https://developercenter.robotstudio.com/api/rwsApi/rwretcode_get_return_codes.html
+     *
+     * \param client RWS client
+     * \param code code to be checked.
+     * 
+     * \return \a RetcodeInfo containing the result.
+     *
+     * \throw \a RWSError if something goes wrong.
+     */
+    RetcodeInfo getRetcodeInfo(RWSClient& client, int const& code);
```

Comment:
Passing an `int` by reference does not really make sense: you pass a pointer to `int`, which is bigger than the `int` itself, and also needs to be dereferenced. The compiler will optimize it out, but what is the purpose?
```suggestion
    RetcodeInfo getRetcodeInfo(RWSClient& client, int code);
```